lib/features/manage_account/presentation/mailbox_visibility/mailbox_visibility_view.dart

```diff
@@ -0,0 +1,196 @@
+import 'package:tmail_ui_user/features/mailbox/presentation/model/mailbox_categories.dart';
+import 'package:tmail_ui_user/features/mailbox/presentation/model/mailbox_node.dart';
+import 'package:tmail_ui_user/features/manage_account/presentation/mailbox_visibility/mailbox_visibility_controller.dart';
+import 'package:tmail_ui_user/features/manage_account/presentation/mailbox_visibility/widgets/mailbox_visibility_folder_tile_builder.dart';
+import 'package:tmail_ui_user/features/manage_account/presentation/mailbox_visibility/widgets/mailbox_visibility_header_widget.dart';
+import 'package:tmail_ui_user/features/manage_account/presentation/menu/settings_utils.dart';
+import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
+
+class MailboxVisibilityView extends GetWidget<MailboxVisibilityController> with AppLoaderMixin {
+  final _responsiveUtils = Get.find<ResponsiveUtils>();
+  final _imagePaths = Get.find<ImagePaths>();
+
+  MailboxVisibilityView({Key? key}) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: _responsiveUtils.isWebDesktop(context)
+          ? AppColor.colorBgDesktop
+          : Colors.white,
+      body: Container(
+        width: double.infinity,
+        margin: _responsiveUtils.isWebDesktop(context)
+            ? const EdgeInsets.all(24)
+            : EdgeInsets.symmetric(horizontal: SettingsUtils.getHorizontalPadding(context, _responsiveUtils)),
+        color: _responsiveUtils.isWebDesktop(context) ? null : Colors.white,
+        decoration: _responsiveUtils.isWebDesktop(context)
+            ? BoxDecoration(
+                borderRadius: BorderRadius.circular(20),
+                border: Border.all(color: AppColor.colorBorderBodyThread, width: 1),
+                color: Colors.white)
+            : null,
+        child: ClipRRect(
+          borderRadius: BorderRadius.circular(
+              _responsiveUtils.isWebDesktop(context) ? 20 : 0),
+          child: Padding(
+            padding: const EdgeInsets.only(top: 24),
+            child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  if (_responsiveUtils.isWebDesktop(context))...[
+                    const MailboxVisibilityHeaderWidget(),
+                    _buildLoadingView(),
+                    const Padding(
+                      padding:  EdgeInsets.symmetric(vertical: 16),
+                      child:  Divider(color: AppColor.colorDividerMailbox, height: 0.5, thickness: 0.2),
+                    ),
+                  ],
+                  Expanded(child: _buildListMailbox(context)),
+                ]
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildLoadingView() {
+    return Obx(() => controller.viewState.value.fold(
+      (failure) => const SizedBox.shrink(),
+      (success) => success is LoadingState
```

Comment:
I did not see the loading indicator, need to wait with a blank page, after longtime, mailboxes are showed. please check it

Reply:
Demo this case 

`Web`

https://user-images.githubusercontent.com/99852347/218651323-bb70f677-0383-4720-8b3c-9a4102ffd307.mov

`Mobile`

[Mobile.webm](https://user-images.githubusercontent.com/99852347/218651489-da49bca8-be9f-48c4-995e-ff417ae9c5a8.webm)